infra/prod/_modules/functions/variables.tf

```diff
@@ -40,3 +40,17 @@ variable "rp_func" {
   })
   description = "Configuration of the relying-party func app"
 }
+
+variable "op_func" {
+  type = object({
+    autoscale_default = number
+    autoscale_minimum = number
+    autoscale_maximum = number
+    app_settings = list(object({
+      name                  = string
+      value                 = optional(string, "")
+      key_vault_secret_name = optional(string)
+    }))
+  })
+  description = "Configuration of the openid-provider func app"
+}
```

Comment:
I was thinking since we are using modules, we can directly write this value into `function_op` file

Reply:
I Agree, but we used a similar approach with [rp-func](https://github.com/pagopa/io-fims/blob/main/infra/prod/_modules/functions/variables.tf#L30), do we want to move that variable in the proper file as well? @lucacavallaro

Reply:
I moved also the "rp_func" variables for consistency [ff3d30b](https://github.com/pagopa/io-fims/pull/27/commits/ff3d30b8b3599f8904f3ca833ed5b24bb20c03d6)